docs/contributing.rst

```diff
@@ -73,6 +73,44 @@ flag to Git commit commands, as in ``git commit --no-verify``.
 .. [#] In rare cases changes to one file might invalidate an unchanged file, such as
    when modifying the return type of a function used in another file.
 
+Benchmarks
+^^^^^^^^^^
+
+PySTAC uses `asv <https://asv.readthedocs.io>`_ for benchmarking. Benchmarks are
+defined in the ``./benchmarks`` directory. Due to the inherent uncertainty in
+the environment of Github workflow runners, benchmarks are not executed in CI.
+If your changes may affect performance, we recommend checking your benchmarks
+locally using the provided script:
```

Comment:
Might be worth being a little more explicit about what `scripts/bench` is doing. Something like:

>If your changes may affect performance, we recommend locally checking your benchmarks against the main branch benchmarks using the provided script.

That may need some wordsmithing.